.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
We can setup a global gitignore in the personal workspace, so there is no need to tell us your favorite ide here.
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer